tests/api.rs

```diff
@@ -168,6 +168,7 @@ mod api_tests {
             .dispatch();
         debug!("{:?}", res.body_string());
         assert_eq!(res.status(), rocket::http::Status::Ok);
+        assert_eq!(res.body_string().unwrap(), r#"{"id":null,"timestamp":"2018-01-01T01:01:01Z","duration":2.0,"data":{}}"#);
```

Comment:
Why is `"id": null`? That's not what aw-server does is it?

Reply:
Because id is not known until the SQLite transaction is performed. The peewee storage can know the id because it actually does the write directly, but that is also why it's so incredibly slow. I haven't checked but I'm pretty certain that the sqlite storage doesn't support that either because of that reason.

Reply:
Oh, I was not aware of this.